### ISSUE TITLE
Edited settings file

### ISSUE THREAD

**Issue by Rene123** [whiteshark-1975/CitiesSkyline2-CheatOverhaul#6]
[settings.json](https://github.com/whiteshark-1975/CitiesSkyline2-CheatOverhaul/files/13702374/settings.json)
 For anyone who doesnt want to go through the settings, this is a quick edit of the settings file.

@whiteshark-1975 If you do not wish for this to be put here, please remove it.

**Comment by Rene123** [comment]
[settings.json](https://github.com/whiteshark-1975/CitiesSkyline2-CheatOverhaul/files/13762641/settings.json)
Whiteshark-Whiteshark_Cheat_Overhaul-0.3.25 version, fully reloaded, and mostly edited to remove costs or polution.
"Upkeep": 50,
"ElectricityConsumption": 0,
"WaterConsumption": 0,
"GarbageAccumulation": 0,
"TelecomNeed": 0,
"AirPollution": 0,
"GroundPollution": 0,
"NoisePollution": 0,

A few I had no clue about what they were doing so didnt touch like the Garbagetruckunloadrate

Incinerationplant - producitonperunit
Firestation - vehicleefficiency
earlydisasterwarningsystem - citydisasterdamagerate
maintenancevehicles option - maintenancerate
fireengine options - extinguisherate and spread

**Comment by Rene123** [comment]
[settings.json](https://github.com/whiteshark-1975/CitiesSkyline2-CheatOverhaul/files/13793565/settings.json)
v3.26

**Comment by fleps** [comment]
Should I assume the `settings.json` the mod creates on first run have ALL DEFAULT values, and I modify just the ones I want to customize?

**Comment by Rene123** [comment]
> Should I assume the `settings.json` the mod creates on first run have ALL DEFAULT values, and I modify just the ones I want to customize?

Yes, on first run the settings will be created, and then you can edit the values in the file.
